git-link.el

```diff
@@ -893,15 +893,19 @@ Defaults to \"origin\"."
 ;;;###autoload
 (defun git-link-homepage (remote)
   "Create a URL representing the homepage of the current
-buffer's GitHub/Bitbucket/GitLab/... repository. The
-URL will be added to the kill ring. If 
-`git-link-open-in-browser' is non-nil also call `browse-url'.
+buffer's GitHub/Bitbucket/GitLab/... repository.  The URL will be
+added to the kill ring.  If `git-link-open-in-browser' is non-nil
+or if you pass the prefix argument of 1 also call `browse-url'.
 
 With a prefix argument prompt for the remote's name.
 Defaults to \"origin\"."
 
   (interactive (list (git-link--select-remote)))
-  (let* (handler remote-info (remote-url (git-link--remote-url remote)))
+
+  (let* (handler remote-info
+		 (remote-url (git-link--remote-url remote))
+		 (git-link-open-in-browser (or git-link-open-in-browser (equal 1 current-prefix-arg))))
```

Comment:
Instead of a prefix arg of 1 what do you think about using a double-prefix (<kbd>Ctrl-u Ctrl-u</kbd>) since that's what `git-link` uses? Makes it more consistent across functions. 

Seems that single prefix is what we want but that prompts for remote which I _guess_ is necessary to keep inline with `git-link` since homepage could be based on remote name.

Reply:
Sure, modified to use double-prefix instead.